### PR TITLE
[artifactory][artifactory-ha] Support custom CA and certificates

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:
Support custom CA and certificates

### CODE REVIEW

File: stable/artifactory-ha/README.md [hunk 1]
```diff
@@ -986,6 +986,36 @@ This will, in turn:
 * Set the shell script we mounted as the `postStartCommand`
 * Copy the `logback.xml` file to its proper location in the `$ARTIFACTORY_HOME/etc` directory.
 
+### Establishing TLS and Adding certificates
+In HTTPs, the communication protocol is encrypted using Transport Layer Security (TLS). By default, TLS between JFrog Platform nodes is disabled.
```
Comment (eldada) [inline stable/artifactory-ha/README.md:990]: `HTTPs` -> `HTTPS`

File: stable/artifactory-ha/README.md [hunk 2]
```diff
@@ -1019,6 +1049,174 @@ helm upgrade --install artifactory-ha -f filebeat.yaml --namespace artifactory-h
 
 This will start sending your Artifactory logs to the log aggregator of your choice, based on your configuration in the `filebeatYml`
 
+### Install Artifactory HA with Nginx and Terminate SSL in Nginx Service(LoadBalancer).
+To install the helm chart with performing SSL offload in the LoadBalancer layer of Nginx.
+For Ex: Using AWS ACM certificates to do SSL offload in the loadbalancer layer.
+
+```bash
+helm upgrade --install artifactory-ha \
```
Comment (eldada) [inline stable/artifactory-ha/README.md:1057]: Can we change this to a yaml block and not command line?

File: stable/artifactory-ha/README.md [hunk 3]
```diff
@@ -1019,6 +1049,174 @@ helm upgrade --install artifactory-ha -f filebeat.yaml --namespace artifactory-h
...
+   --set nginx.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-ssl-cert"="arn:aws:acm:xx-xxxx:xxxxxxxx:certificate/xxxxxxxxxxxxx" \
+   --set nginx.service.annotations."service\.beta\.kubernetes\.io"/aws-load-balancer-backend-protocol=http \
+   --set nginx.service.annotations."service\.beta\.kubernetes\.io"/aws-load-balancer-ssl-ports=https \
+   --namespace artifactory-ha center/jfrog/artifactory-ha
+```
+
+### Ingress and TLS
+To get Helm to create an ingress object with a hostname, add these two lines to your Helm command:
+```bash
+helm upgrade --install artifactory-ha \
```
Comment (eldada) [inline stable/artifactory-ha/README.md:1069]: Yaml block please

File: stable/artifactory-ha/README.md (hunk 1 above)
Comment (chukka) [reply stable/artifactory-ha/README.md:990]: Done

File: stable/artifactory-ha/README.md (hunk 2 above)
Comment (chukka) [reply stable/artifactory-ha/README.md:1057]: Done

File: stable/artifactory-ha/README.md (hunk 3 above)
Comment (chukka) [reply stable/artifactory-ha/README.md]: Done

File: stable/artifactory-ha/README.md (hunk 2 above)
Comment (chukka) [reply stable/artifactory-ha/README.md]: Done

**Pull request merged**